config/samples/networking_v1alpha1_ratelimit.yaml

```diff
@@ -3,7 +3,15 @@ kind: RateLimit
 metadata:
   name: ratelimit-sample
 spec:
-  # Add fields here
-  ratelimitconfig:
-    Data:
-      field: test
+  targetRef:
+    apiVersion: "networking.istio.io/v1alpha3"
+    kind:       VirtualService
+    name:       noodle-vs
+    namespace:  noodle-v1
+  destinationCluster: "outbound|80||chicken-head-nginx.chicken-head.svc.cluster.local"
```

Comment:
What this exactly mean? Is not this configured in the virtual service or in any other config?

Reply:
In the Virtual service could be one or more istio clusters.
For instance, in the noodle VS there are 2 

server.noodle-v1.svc.cluster.local
server-whitelist.noodle-v1.svc.cluster.local